include/cross.h

```diff
@@ -127,4 +128,9 @@ void close_directory(dir_information* dirp);
 FILE *fopen_wrap(const char *path, const char *mode);
 FILE *fopen_wrap_ro_fallback(const std::string &filename, bool &is_readonly);
 
+bool wild_match(const char *haystack, const char *needle);
+bool WildFileCmp(const char * file, const char * wild, bool long_compare = false);
+
+bool get_expanded_files(const std::string &path, std::vector<std::string> &files, bool files_only, bool skip_native_path = false) noexcept;
```

Comment:
Can you run clang format on this commit? 
It should nicely stack the arguments for this long function, each left-aligned.

Reply:
It is true that I missed the clang-format on this header file earlier. Done.